src/pages/workspaces/workspace/Data.js

```diff
@@ -574,7 +588,10 @@ const WorkspaceData = _.flow(
   const [uploadingFile, setUploadingFile] = useState(false)
   const [entityMetadataError, setEntityMetadataError] = useState()
   const [snapshotMetadataError, setSnapshotMetadataError] = useState()
-  const [sidebarWidth, setSidebarWidth] = useState(280)
+  const [sidebarWidth, setSidebarWidth] = useState(340)
```

Comment:
I need to check with Ean to make sure this is okay, but the addition of the green "pills" when in search-mode made it a bit cramped with the previous default width

Reply:
I reverted this in 6473444e567975c124f4fe01998adf9a6b716bae

We decided to go with truncating long pill numbers, so showing (100141) as (100k), etc